scripts/normalize-yaml

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+$LOAD_PATH.unshift(File.expand_path('../lib', File.dirname(__FILE__)))
+require 'yaml_normalizer'
+require 'yaml'
+
+YamlNormalizer.run(ARGV)
```

Comment:
If this script is meant to be run every time we receive new translations, I think it would be better to give it more visibility, whether in a Wiki or some other place where developers are likely to look up translation-related documentation. What do you think? At the very least, the example in the spec should mention that the command needs a path passed after it. Running `scripts/normalize-yaml` by itself doesn't do anything. Also, I'm assuming you can give the script a directory, and it will normalize all the files in the directory, but when I tried `scripts/normalize-yaml config/locales`, it gave me an error. How do you pass in a directory?